companies/pinterest.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Pinterest
+tags: ["tech", "social media"]
+image: NewsroomPost-Header-200602_0.jpg
+blm_statements:
+  - url: https://newsroom.pinterest.com/en/post/listening-and-acting
+    date_posted: 2020-06-02
+    type: Blog post >
+        Hello.
```

Comment:
Oops, I think you meant to put this in the `raw_text` field.

Have you been able to run the app locally and open the Pinterest page to see it in action?  It should be at `localhost:8000/companies/pinterest` once you have the server running on your machine.

Reply:
I couldn't because of an error with `husky`, now after googling the error and fixed it by installing `husky@next`. :thinking:

Reply:
fixed the issue and checked the page by running it local this time :smile: